p01-data-binding/src/Content/Content.js

```diff
@@ -1,33 +1,58 @@
+  Why do we use it?
+  It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+  Where does it come from?
+  Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
+  The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.`
+  const [title, setTitle] = useState("Title")
+  const [content, setContent] = useState(defaultContent)
+  const [editing, setEditing] = useState(false)
+  const [displayText, setDisplayText] = useState("Edit")
+
+  const handleTitleChange = (e) => {
+    setTitle(e.target.value)
+  }
+
+  const handleContentChange = (e) => {
+    setContent(e.target.value)
+  }
+
+  const handleDisplayClick = (e) => {
+    setEditing(!editing)
+    if (editing)
+      setDisplayText("Edit")
+    else
+      setDisplayText("Done")
+  }
+
+  const Editor = () => {
+    return(
       <div className="card editor">
         <div className="form-group">
           <label>Title</label>
-          <input type="text" />
+          <input type="text" value={title} onChange={handleTitleChange}/>
         </div>
         <div className="form-group">
           <label>Description</label>
-          <textarea rows={7} />
+          <textarea rows={7} value={content} onChange={handleContentChange}/>
         </div>
       </div>
+    );
+  }
+
+  return (
+    <div className="container">
+      { editing && Editor() }
```

Comment:
You can call the component as a function, because at the end of the day a Function component is a function
```
Editor()
```
but you can also call it like a tag, this way is more readable that you are rendering a component
```
{ editing && <Editor /> }
```